src/bridge/qos.go

```diff
@@ -4,13 +4,21 @@ import (
 	"errors"
 	"fmt"
 	log "minilog"
-	"strconv"
 )
 
 // Used to calulate burst rate for the token bucket filter
 const KERNEL_TIMER_FREQ uint64 = 250
 const MIN_BURST_SIZE uint64 = 2048
 
+type QosParams struct {
```

Comment:
I agree, this was mostly to avoid reconverting back to a string before passing the arguments as part of a call to processWrapper() but that doesn't make sense outside qos being used through the command line interface.